modules/openapi-generator/src/test/java/org/openapitools/codegen/haskellservant/HaskellModelTest.java

```diff
@@ -24,6 +24,5 @@
 
     @Test(description = "convert a haskell model with dots")
     public void modelTest() {
-        Assert.assertEquals(true, true);
```

Comment:
this whole (test) class seems to be obsolete then?

Reply:
Good catch. We will revise it with better model tests for the Haskell Servant generator later.